types/adobe__node-fetch-retry/package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
please remove this file, it's just empty artefact

Reply:
Removed in 68d0395.